src/components/container/BasicPointsForBadges/BasicPointsForBadges.stories.tsx

```diff
@@ -8,22 +8,31 @@ import DailyDataGoal from "../DailyDataGoal";
 import { Grid, Title } from "../../presentational";
 import BasicBadges from "../BasicBadges";
 import Divider from "../../presentational/Divider";
+import { Description } from "@storybook/blocks"
+import { Meta, StoryObj } from "@storybook/react";
+import { ConnectExternalAccountActivity, CustomActivity, SurveyCompletionActivity } from "../../../../dist";
```

Comment:
These should be imported from `../../../helpers`.

The `DailyDataActivity` import can also be simplified to `../../../helpers`.

All of them can be combined with the existing import for `DailyDataType`.